ce/api/util.py

```diff
@@ -5,6 +5,7 @@
 import numpy.ma as ma
 from netCDF4 import Dataset
 from shapely.wkt import loads
+from modelmeta import *
```

Comment:
Yeah, I don't *normally* like to do this, but we traverse across 7 ORM classes in the search, and that felt too verbose for me.